backend/src/services/model.ts

```diff
@@ -21,21 +21,32 @@ export function checkModelRestriction(model: ModelInterface) {
   }
 }
 
-export type CreateModelParams = Pick<ModelInterface, 'name' | 'teamId' | 'description' | 'visibility' | 'settings'>
+export type CreateModelParams = Pick<
+  ModelInterface,
+  'name' | 'teamId' | 'description' | 'visibility' | 'settings' | 'collaborators'
```

Comment:
Could you also add 'kind' to this list while you're here? I think I accidentally removed it at some point...